AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -981,6 +981,7 @@ private static void applyContentInResponse(JSONObject response, boolean alwaysAp
         || !messages.equals(VarCache.getMessageDiffs())
         || !updateRules.equals(VarCache.getUpdateRuleDiffs())
         || !eventRules.equals(VarCache.getEventRuleDiffs())
+        || !variants.equals(VarCache.variants())
```

Comment:
Can you give me quick context why we need this variants? Also, any reasons, we dont have test cases for this?